### PR TITLE
Updates docs builder to current version

### PR DESCRIPTION
**Pull request by goodroot**

### CODE REVIEW

**Review by gtback — CHANGES_REQUESTED** [review]

File: .github/workflows/dev-docs-builder.yml [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
-      - '**.gif'
-    types: [closed, opened, synchronize, reopened]
+      - 'docs/**.docapi.json'
+      - 'docs/**.devdocs.json'
+      - 'docs/**.jpg'
+      - 'docs/**.jpeg'
+      - 'docs/**.png'
+      - 'docs/**.svg'
+      - 'docs/**.gif'
+    types: [opened, synchronize]
```
Comment (goodroot) [inline .github/workflows/dev-docs-builder.yml:16]:
```suggestion
    types: [closed, opened, synchronize]
```

File: .github/workflows/dev-docs-builder.yml [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
-      - '**.docapi.json'
-      - '**.devdocs.json'
-      - '**.jpg'
-      - '**.jpeg'
-      - '**.png'
-      - '**.gif'
-    types: [closed, opened, synchronize, reopened]
+    # Change docs dir to your repos docs dir
+      - 'docs/**.mdx'
+      - 'docs/**.docnav.json'
```
[the remark this answers is not in the document]
Comment (goodroot) [reply .github/workflows/dev-docs-builder.yml]: Good catch!

**Review by gtback — APPROVED** [review]

**Comment by goodroot** [comment]
@gtback Woohoo!

You gotta merge. :)

**Comment by gtback** [comment]
@goodroot I just gave you write permission to this repo, but I'll merge it anyway.

**Pull request merged**